package materialshader;

message Binding
{
  optional string texture = 1;
  optional string uniform = 2;
}

message MaterialShader
{
  repeated Binding textures = 1;
  optional bool has_light = 2;
  optional bool has_color = 3;
}
